PWGCF/MultiparticleCorrelations/Core/MuPa-MemberFunctions.h

```diff
@@ -499,6 +558,150 @@ void BookWeightsHistograms()
 
 //============================================================
 
+void BookNestedLoopsHistograms()
+{
+  // Book all nested loops histograms.
+
+  // a) Book the profile holding flags;
+  // *) ...
+
+  if (fVerbose) {
+    Green(__PRETTY_FUNCTION__);
```

Comment:
I don't know what `Green` does but, please, for logging you have to use the O2 logging machinery

Reply:
"Green" is a simple utility function, which prints its arguments in green, this is how it's implemented:

`void Green(const char* text)
{
  cout << "\n\033[1;32m" << text << "\033[0m\n"
       << endl;
}`

It uses the standard escape sequences for colors(*), recognized by other languages as well. This gives me some uniformity when I have to implement the same thing across different languages. 

(*) 
`Black        0;30     Dark Gray     1;30
Red          0;31     Light Red     1;31
Green        0;32     Light Green   1;32
Brown/Orange 0;33     Yellow        1;33
Blue         0;34     Light Blue    1;34
Purple       0;35     Light Purple  1;35
Cyan         0;36     Light Cyan    1;36
Light Gray   0;37     White         1;37`